app/views/static_pages/about_us.html.erb

```diff
@@ -34,10 +34,11 @@
 
       <h2 class="fontsize-h3"><%= _('Getting started') %></h2>
 
-        <p><%= _("If you have an account please sign in and start creating or editing your DMP.") %></p>
+        <p><%= _("If you have an account, please sign in and start creating or editing your DMP.") %></p>
+        <!--
```

Comment:
Sorry, why are we commenting out this next part? If we don't need it we should remove it, but it looks like this code is something we do want here.

Reply:
All good. This related to #407. Currently, the "Sign Up" and "Help" messages are repeated in the "Getting Started" section of the "About Us" page https://assistant.portagenetwork.ca/about_us.

I'm thinking we'll need to remove one of the "Sign Up" messages and one of the "Help" messages. But I actually prefer the code I commented out in comparison to what I kept. However, translations aren't currently available for the code that I do prefer.

Reply:
We can double-check with @isawil to see which copy we like best. We can leave just one instead of waiting for the translation for the missing piece.

Does this make sense?

Reply:
Cool, yes, that makes sense. I'll verify which one we should keep.